src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -278,6 +279,23 @@ private static void EnsureSettingsNode(XmlDocument settings, TestRunSettings set
             return null;
         }
 
+        /// <summary>
+        /// Testplatform.Fakes has version 16.0 in Dev 16
+        /// and version 17.0 in Dev 17.
+        /// </summary>
+        /// <returns></returns>
+        private static Assembly LoadTestPlatformAssembly()
+        {
+            try
+            {
+                return Assembly.Load(new AssemblyName(FakesConfiguratorAssemblyDev16));
```

Comment:
If we know where the Fakes assembly is on disk always, we could use Assembly.LoadFrom and skip the version altogether...

Reply:
I didn't know that the API existed. Giving it a try, I found out that the API doesn't exist for .NetStandard 1.3 that the project builds for, so we will have to use this API